frontend/test/programs/ntu-uaf.c

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
Might be good to run `clang-format` on this (see #44)

Reply:
Done.